Doc/library/curses.rst

```diff
@@ -68,6 +68,20 @@ The module :mod:`curses` defines the following exception:
 The module :mod:`curses` defines the following functions:
 
 
+.. function:: assume_default_colors(fg, bg)
+
+   Allow use of default values for colors on terminals supporting this feature.
+   Use this to support transparency in your application.
+
+   Assign terminal default foreground/background colors to color number ``-1``.
+   So ``init_pair(x, COLOR_RED, -1)`` will initialize pair *x* as red
+   on default background and ``init_pair(x, -1, COLOR_BLUE)`` will
+   initialize pair *x* as default foreground on blue.
+   Change the definition of the color-pair ``0`` to ``(fg, bg)``.
```

Comment:
I think we should just keep "Change the definition of the color 0 to (fg, bg)". And let's mention that (-1, -1) is equivalent to "use_default_colors()".

Reply:
This is more than that. The main effect of use_default_colors() and assume_default_colors() is that they enable using the terminal default foreground/background colors (as color -1). Changing the color-pair 0 is a secondary effect, and I wonder why they did not simply make init_pair(0, fg, bg) working.

use_default_colors() is just assume_default_colors(-1, -1).

Reply:
This was a copy from the ncurses documentation. That sentence is an explanation of the previous sentence, "Assign terminal default foreground/background colors to color number -1."

Change the definition of the color-pair 0 is different effect. Maybe move it to a separate paragraph? Or even use an enumeration list?